tests/dummy/app/templates/demo/accordion.hbs

```diff
@@ -1,61 +1,58 @@
 <div class="bs-example">
   <!-- BEGIN-SNIPPET accordion-demo -->
-  {{#bs-accordion as |acc|}}
-    {{#acc.item value="1" title="First item"}}
-      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur ea eos hic neque quia sequi sunt. Amet at atque corporis earum et fugiat illum magnam nisi sapiente voluptatem. Laudantium, quam?</p>
-    {{/acc.item}}
-    {{#acc.item value="2" as |aitem|}}
-      {{#aitem.title}}
-        <strong>Second</strong> <i>item</i>
-      {{/aitem.title}}
-      {{#aitem.body}}
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur ea eos hic neque quia sequi sunt. Amet at atque corporis earum et fugiat illum magnam nisi sapiente voluptatem. Laudantium, quam?</p>
-      {{/aitem.body}}
-    {{/acc.item}}
-    {{#acc.item value="3" title="Third item"}}
-      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur ea eos hic neque quia sequi sunt. Amet at atque corporis earum et fugiat illum magnam nisi sapiente voluptatem. Laudantium, quam?</p>
-    {{/acc.item}}
-  {{/bs-accordion}}
+  <BsAccordion as |acc|>
+    <acc.item value="1" @title="First item">
+      <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur ea eos hic neque quia sequi sunt. Amet at atque corporis earum et fugiat illum magnam nisi sapiente voluptatem. Laudantium, quam?  </p>
+    </acc.item>
+    <acc.item value="2" as |aitem|>
+      <aitem.title>
+        <strong> Second </strong>
```

Comment:
These additional white spaces look weird, I guess they were introduced by the AST transform?

Haven't (yet) looked deeply into your codemod, are you using https://github.com/ember-template-lint/ember-template-recast for it?

Reply:
Yes, it was due to the codemod, actually i am using prettier at the final stage to pretty print the templates using the glimmer parser. My original idea was to use the ember-template-recast but it has got some issues with the current version
https://github.com/ember-template-lint/ember-template-recast/issues/16

Reply:
Can you remove those manually?